specs/http-api/README.md

```diff
@@ -323,18 +367,39 @@ Uses DID authn via Bearer token in header.
 ## List Exchanges
 
 ### Description
-Returns an array of tbdex message arrays
-
-### Authentication
-Uses DID authn via Bearer token in header.
+Returns an array of tbdex message arrays (a list of exchanges)
 
 ### Endpoint
 `GET /exchanges`
 
+### Authentication
+A tbdex http client must send an Http Auth Header in the form of a JWT `Bearer` token.
+
+The JWT must consist of JWS header and JWT payload containing the below fields, and signed by the client's DID.
```

Comment:
Aligning with the terminology as defined in https://www.rfc-editor.org/rfc/rfc7519#section-2. 

Also making it clear how the signing happens. It sounds super confusing, so may I could see it being separated into another section (or simply removed)

```suggestion
The JWT must consist of JOSE header and JWT Claims Set containing the below fields, and signed using the verification material contained in a verification method that's published as the authentication verification relationship within the client's published DID document.
```

Reply:
thank you! that's quite thorough, but also a bit confusing to read to me too. i'll look into how i can re-work this.